components/booking/bookingsFilter.tsx

```diff
@@ -1,30 +1,49 @@
 import Card from '@mui/material/Card'
 import CardContent from '@mui/material/CardContent'
 import { 
+  Checkbox,
   FormControl,
   InputLabel,
   MenuItem,
   Select,
   SelectChangeEvent
 } from '@mui/material'
 
-function BookingFilter({dataUpdateParams}) {
+function BookingsFilter({dataUpdateParams, teamMembers}) {
   /**
    * dataUpdateParams contains:
    *  - setData -> needed to reset the data
    *  - setHasMore -> set it "true" when the data is reset
    *  - setOrderBy -> needed to update the data sort type
    *  - setSortOrder -> needed to update the data order (ascending or descending)
+   *  - selectedTeamMembers -> necessary to know if the team member has been 
+   *    selected and should be deselected or vice versa
+   *  - setSelectedTeamMembers -> necessary to update the list with the selected 
+   *    ones
    */
 
+  const selectedTeamMembers = dataUpdateParams.selectedTeamMembers
+  const setSelectedTeamMembers = dataUpdateParams.setSelectedTeamMembers
+  const teamMembersNames = teamMembers.map(({ name }) => name)
```

Comment:
shouldn't we do this inside a useEffect?

Reply:
if we use "useEffect" here "Select All" won't work anymore